Helper/FeatureSwitch/Definitions.php

```diff
@@ -296,12 +296,17 @@ class Definitions
      * Enable shopper assistant
      */
     const M2_ENABLE_SHOPPER_ASSISTANT = 'M2_ENABLE_SHOPPER_ASSISTANT';
-    
+
     /**
      * Skip cart discount total mismatch
      */
     const M2_SKIP_CART_DISCOUNT_TOTAL_MISMATCH = 'M2_SKIP_CART_DISCOUNT_TOTAL_MISMATCH';
 
+    /**
+     * Enable API driven cart integration
+     */
+    const M2_ENABLE_API_DRIVEN_CART_INTEGRATION = 'M2_ENABLE_API_DRIVEN_CART_INTEGRATION';
```

Comment:
Let's discuss the best name with the team.
This one can be a little bit confusing.

Reply:
Let's rename it to M2_FETCH_CART_VIA_API

Reply:
Renamed